components/06-examples/machtigen.hbs

```diff
@@ -0,0 +1,78 @@
+{{> '@header' }}
+<main>
+  <div>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Inloggen</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <ul class="nav nav-tabs">
+              <li class="active">
+                <a href="machtigen.html">Voor mezelf</a>
+              </li>
+              <li >
+                <a href="machtigen1.html">Voor een ander</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="row dso-rich-content">
+          <div class="col-sm-12">
+            <div class="dso-highlight-box dso-border" style="margin: 16px 0px 8px 0px"> 
+              <div class="row">
+                <div class="col-md-2 col-sm-2  col-xs-4 text-center"><img src="{{path '/images/Digid.png' }}" alt="Digid Logo" style="width:60px; height:60px;"></div>
+                <div class="col-md-10 col-sm-4 col-xs-4 ">
+                  <button type="button" class="btn btn-link" >
+                    <span>Inloggen met digID</span><svg class="di di-angle-right">
+                    <use href="../../dso-icons.svg#angle-right" />
+                    </svg>
+                  </button>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-sm-12">
+            <div class="dso-highlight-box dso-border" style="margin: 8px 0px 8px 0px">
+              <div class="row">
+                <div class="col-md-2 col-sm-2 col-xs-4 text-center" ><img src="{{path '/images/Eherkenninglogo.png' }}" alt="Eherkenning logo" style="width:60px; height:60px;"></div>
```

Comment:
Bovenstaande opmerkingen gelden ook voor dit item. Aangezien je hier markup dupliceert waar alleen de content anders van is, waarom maak je er geen data object van binnen de yaml en itereer je over deze data in de `hbs` file?